d2bs/kolbot/libs/bots/DiabloHelper.js

```diff
@@ -72,6 +73,43 @@ function DiabloHelper() {
 		this.infLayout = this.getLayout(392, 7893); // 1 = "I", 2 = "J"
 	};
 
+	this.openSeal = function (classid) {
```

Comment:
As great as this is, isn't this out of the scope of adding the seal order to Diablo.js?

Also its an exact copy of Diablo's open seal functions, maybe we can combine this?

Reply:
scope creep for sure, yep. I'll pull this out and create a new pr. I'll see what I can do about combining too

Reply:
actually, I take that back. It's in scope because if the helper does a seal on his own the leader will still have to go pop it, unless the helper can open seals.